src/kakarot/accounts/eoa/library.cairo

```diff
@@ -90,7 +82,7 @@ namespace ExternallyOwnedAccount {
         bitwise_ptr: BitwiseBuiltin*,
         range_check_ptr,
     }(call_array_len: felt, call_array: CallArray*, calldata_len: felt, calldata: felt*) -> () {
-        if (call_array_len == 0) {
+        if (call_array_len == FALSE) {
```

Comment:
Imo in this scenario:
`call_array_len == 0` is preferable over `call_array_len == FALSE`,
As we're actually checking what the call_array length is and not whether it is true or false.